libs/sdk-model/api/sdk-model.api.md

```diff
@@ -505,7 +513,7 @@ export interface IAttributeDescriptorBody {
 // @public
 export interface IAttributeDisplayFormMetadataObject extends IMetadataObject {
     attribute: ObjRef;
-    displayFormType?: string;
+    displayFormType?: AttributeDisplayFormType | string;
```

Comment:
String here is just for backward compatibility reason now (but it's not needed anymore),  is it correct? Or do we have still some display form types that are not covered by that enum?

Reply:
Bear model is using string type too. So not sure what surprises can come from MD to converter and need to be converted to this type. But main motivation was backward compatibility and I dont know any widely used type not covered by enum